playwright/tests/organize/people/views/view-detail.spec.ts

```diff
@@ -7,7 +7,12 @@ import AllMembersRows from '../../../../mockData/orgs/KPD/people/views/AllMember
 import KPD from '../../../../mockData/orgs/KPD';
 import NewView from '../../../../mockData/orgs/KPD/people/views/NewView';
 import NewViewColumns from '../../../../mockData/orgs/KPD/people/views/NewView/columns';
-import RosaLuxemburg from '../../../../mockData/users/RosaLuxemburg';
+
+const NewPerson = {
+    first_name: 'New',
+    id: 1337,
+    last_name: 'Person',
+};
```

Comment:
I think that it makes more sense to have mock data that is meant to be more like "real" data, rather than more contrived examples. So rather than make a generic "New Person", do you think it would be a good idea to make a new real person, and add them to the view? I think that simulates a more realistic example, and it is nice to have this data in a shared place for the next time we need a new person

Reply:
I realise I definitely failed to do this in the past, but that was why I made the folder structure where we can add multiple people and stuff like that.

Reply:
I agree! I should have commented on this or described it better, but I actually think we need to refactor the mock data a bit. There is confusion in the data about "user" and "person", which is not a disaster because so far we've only cared about the fields that happen to follow the same schema (`id`, `first_name` and `last_name`). But at some point we will start running into problems with this confusion, and properly fixing that would be a refactor that would affect multiple tests. So I opted to not try to solve that in this PR, but keep it for later. I should probably create an issue though.

Here's my proposed actions:

1. Leave this code as it is in this PR
2. Create an issue about refactoring the mock data to use user/person correctly
3. Add a `TODO` comment here so it's not forgotten

Do you think that makes sense?

Reply:
alrighty yep!